build/xtask/src/config.rs

```diff
@@ -344,6 +344,10 @@ impl Config {
             "riscv32imc-unknown-none-elf" | "riscv32imac-unknown-none-elf" => {
                 MpuAlignment::Chunk(16)
             }
+            // Qemu on RV64 assumes entire page(4KiB) has the same access permissions
```

Comment:
I think we may, in the future, want to break things out based on a configuration. e.g. s-mode/u-mode without pmps, m-mode/u-mode with pmps, etc. That would then allow a better represenation of alignment requirements. And it's not really a QEMU requirement.

Reply:
Agreed. We will need some configs around this to decide how to handle the different requirements for different cases.

With respect to qemu, there is a patch that fixes the current behavior: https://lists.nongnu.org/archive/html/qemu-devel/2022-09/msg01445.html